gradle/publishing.gradle

```diff
@@ -2,6 +2,12 @@ apply plugin: 'maven-publish'
 
 task sourceJar(type: Jar) {
     classifier = 'sources'
+    from sourceSets.main.allJava
+}
+
+task delombokedSourceJar(type: Jar, dependsOn: classes) {
+    classifier = 'sources-delomboked'
```

Comment:
does it make sense to publish it at all? I guess no IDE support alternative source JARs

Reply:
It is a good question. I doubt any IDE has an automatic way, for sure.

My only reasoning for keeping this at all is to give people a manual workaround, should we find that there is a debugging case where the delomboked JAR is better. I'm not sure if that would be needed, though, so I'd be happy to drop it.

Reply:
Since IDEA (I assume Eclipse does this too) supports decompiling, it kinda gives the delomboked sources.

Given how hard it would be to attach sources with a custom classifier, I would say dropping it is okay 👍